doc/RelNotes.md

```diff
@@ -0,0 +1,53 @@
+Servus API Documentation {#mainpage}
+=========
+
+[TOC]
+
+# Introduction {#Introduction}
+
+Welcome to Servus, a small C++ network utility library that provides a zeroconf
+API, uri parsing and UUIDs.
+
+Servus 0.1 can be retrieved by cloning the [source
+code](https://github.com/HBPVIS/servus). Please file a [Bug
+Report](https://github.com/HBPVis/servus/issues) if you find any issues with
+this release.
+
+## Features {#Features}
+
+Servus provides classes for:
+
+* 128 bit UUIDs.
+* A URI class to parse strings using generic syntax from [RFC3986](https://www.ietf.org/rfc/rfc3986.txt).
+* Zeroconf announcing and browsing using Avahi or DNSSD. This is an optional
+  feature available only if the dependencies are found.
+
+- - -
+
+## Changes {#Changes}
+
+### Version 0.1
+
+* Initial version.
+
+- - -
+
+# About {#About}
+
+Servus is partially cross-platform, the only mandatory dependency is using
+a C++11 compiler. Zeroconf will be available in those platforms were either
```

Comment:
Nevermind, the (rng) implementation seems to need it. As discussed, we'll #ifdef with boost if the need for C++03 arises.